lib/fedora_migrate/datastream_verification.rb

```diff
@@ -0,0 +1,43 @@
+module FedoraMigrate::DatastreamVerification
+  
+  attr_accessor :datastream
+
+  def valid? datastream=nil
+    @datastream = datastream || source
```

Comment:
it's in the mover. This is a module that's added in. I opted for a module, as opposed to a new class, because it also needs target, both of which are declared in the mover and didn't want to have to re-initialize those things all over again.